Adding further advice to get access to DC

danielsollondon opened the pull request:
Should you have lost access to your Windows Domain controller, you will need to restore from a domain controller backup.

azurecla commented:
Hi __@ms-prkhad__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

tamram commented:
#sign-off

Commit:
Adding further advice to get access to DC

Should you have lost access to your Windows Domain controller, you will need to restore from a domain controller backup.
